bungiesearch/management/commands/clear_index.py

```diff
@@ -0,0 +1,34 @@
+import sys
+from optparse import make_option
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+from django.utils import six
+
+
+class Command(BaseCommand):
+    help = 'Clears the search index of its contents.'
+
+    option_list = BaseCommand.option_list + (
+        make_option('--noinput',
```

Comment:
Given how dangerous this is, what I tend to do have have two flags, usually the second one being `--guilty-as-charged`. It sounds a bit dumb and a bit of an overkill, but it saves data: https://github.com/Sparrho/bungiesearch/blob/master/bungiesearch/management/commands/search_index.py#L43 .